example-transports.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
```

Comment:
now we need an examples folder I think

examples/basic.js
examples/transports.js